projects/plugins/jetpack/_inc/lib/core-api/wpcom-fields/theme-fields-is-block-theme.php

```diff
@@ -96,6 +96,9 @@ public function get_schema() {
 	}
 }
 
-// TODO: If this field is also added to Core, add a conditional to load only when using an older version of WordPress.
+// `is_block_theme` was added to the v2/themes endpoint in WordPress 6.3.
 // See https://core.trac.wordpress.org/ticket/58123
-wpcom_rest_api_v2_load_plugin( 'WPCOM_REST_API_V2_Theme_Fields_Is_Block_Theme' );
+global $wp_version;
+if ( version_compare( $wp_version, '6.3', '<' ) ) {
```

Comment:
Do we need to care about 6.3 Beta as well?

Reply:
With 6.3 out now, let's not worry about the beta.